### ISSUE TITLE
Accessibility: Screenreader doesn't react to new private chat

### ISSUE THREAD

**Issue by PhMemmel** [bigbluebutton/bigbluebutton#12845]
**Describe the bug**
Screenreader doesn't announce that there is a private chat popping up. It only triggers on second message. In addition to that: Singular message is announced if count >0 instead of plural message.

**To Reproduce**
1. Join with 2 users in a session
2. Enable screenreader (e. g. nvda) for user1.
3. User2 writes message to user1
4. Private Chat pops up for user1 (with notification of 1 unread message), screenreader doesn't recognize it, though

**Expected behavior**
There should be an announcement that there is a new chat with a new message.

**Actual behavior**
Nothing is being triggered for the screenreader.

**BBB version:**
2.4-alpha2

**Desktop (please complete the following information):**
Tested with current Chrome and FF

**Additional context**
Notifications aren't being read at all on 2.3.8, neither public nor private chat.

Will send a PR for this. Just need a little bit more time for testing and feedback before.

**Comment by PhMemmel** [comment]
#12854 should fix the problem of the first message not being announced in `develop`

However, I can't solve the problem in 2.3.x that follow up messages are not being announced as I don't have a proper 2.3.x-dev-environment right now and can't find the problem just from code review. Thus, backporting #12854 to 2.3.x would fix only part of the problem there.
